scripts/x-charts-pro.exports.json

```
@@ -322,5 +322,10 @@
   { "name": "useYScale", "kind": "Function" },
   { "name": "useZColorScale", "kind": "Function" },
   { "name": "ZAxisContextProvider", "kind": "Function" },
-  { "name": "ZAxisContextProviderProps", "kind": "TypeAlias" }
+  { "name": "ZAxisContextProviderProps", "kind": "TypeAlias" },
+  { "name": "ZoomContext", "kind": "Variable" },
+  { "name": "ZoomData", "kind": "TypeAlias" },
+  { "name": "ZoomOptions", "kind": "TypeAlias" },
+  { "name": "ZoomProps", "kind": "TypeAlias" },
+  { "name": "ZoomProvider", "kind": "Function" }
```

What about moving them in a `.types.ts` files such that we only export types

```suggestion
  { "name": "ZoomData", "kind": "TypeAlias" },
  { "name": "ZoomOptions", "kind": "TypeAlias" },
  { "name": "ZoomProps", "kind": "TypeAlias" },
```